src/ex15_js-oop/task-02.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+class Device {
+  constructor(name, powerCons) {
+    this.name = name;
+    this.powerCons = powerCons;
+    this.isPlugged = false;
+  }
+}
+class HomeAppliance extends Device {
+  constructor(name, powerCons) {
+    super(name, powerCons);
+  }
+}
+class ElectronicDevice extends Device {
+  constructor(name, powerCons) {
+    super(name, powerCons);
+  }
+}
+class Room {
+  constructor() {
+    this.devices = Array.prototype.slice.call(arguments);
+  }
+  plugDevice(io) {
+    const plugDevices = Array.prototype.slice.call(arguments, 1);
+    for (let i = 0; i < plugDevices.length; i++) {
+      for (let j = 0; j < this.devices.length; j++) {
+        if (this.devices[j].name === plugDevices[i].name) this.devices[j].isPlugged = !!io;
```

Comment:
`io` ????

Reply:
its so strange

Reply:
It's a parameter determining whether a device is pluged **i**n or **o**ut

Reply:
the same notice - more clear 
isTurnOn, isEnable or different

Reply:
I like "isTurnOn" better, thanks.